improver_tests/calibration/rainforests_calibration/test_ApplyRainForestsCalibrationLightGBM.py

```diff
@@ -295,11 +297,12 @@ def test_make_decreasing():
 
 
 def test__calculate_error_probabilities(
-    ensemble_features, ensemble_forecast, dummy_lightgbm_models
+    ensemble_features, ensemble_forecast, plugin_and_dummy_models
 ):
-    """Test calculation of error probability cube when using lightgbm Boosters."""
-    plugin = ApplyRainForestsCalibrationLightGBM(model_config_dict={})
-    plugin.tree_models, plugin.error_thresholds = dummy_lightgbm_models
+    """Test calculation of error probability cube when using treelite Predictors."""
```

Comment:
If I understand correctly, the plugin is now parametrized and so this tests both treelite and lightgbm models. So I think the docstring is incorrect.

Reply:
Agreed, this docstring got brought over from the treelite file and not updated.

Reply:
I've done this in e991b76.